SDCTA/src/components/Footer.css

```diff
@@ -0,0 +1,67 @@
+/* Footer Styling */
```

Comment:
We probably want body.tsx, footer.tsx, body.css, etc. to be in one folder called LandingPage .

Reply:
agreed